core/src/main/java/co/aikar/commands/CommandCompletions.java

```diff
@@ -71,6 +72,27 @@ public CommandCompletionHandler registerAsyncCompletion(String id, AsyncCommandC
         return this.completionMap.put("@" + id.toLowerCase(), handler);
     }
 
+    public CommandCompletionHandler setDefaultCompletion(String id, Class... classes) {
+        // Since you don't have to specify the @ with the id anywhere else
+        if(!id.startsWith("@")) {
```

Comment:
This is not desirable. non @ prefixed completions are valid, they are hard coded.

Reply:
Alright, I have removed that.